pyts2/tsformats/imageio.py

```diff
@@ -3,21 +3,36 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from pyts2.utils import *
+from pyts2.tsformats.base import *
+
 import imageio
 import numpy as np
 import rawpy
+import datetime as dt
```

Comment:
are we still doing this (with numpy as np and datetime as dt)?

for numpy I think it is common convention but I havent frequently seen it used with datetime.

Should we be importing just datetime? `datetime.datetime` is clunky but I think preferable to `dt.datetime()`

Reply:
Numpy definitely. dt for datetime is an old, bad habit of mine from the old codebase. We can decide going forward that it's not OK if you want, and I'll do a PR to fix it across the codebase. I'll let you decide. 

for the record, the following are pretty common:
```python
import numpy as np
import scipy as sp
import pandas as pd
import matplotlib.pyplot as plt
import skimage as ski
```

There's probably more, but that's all off the top of my head. I suggest having this as a whitelist of abbreviated imports and using full imports otherwise.